services/cd-service/pkg/repository/repository.go

```diff
@@ -1402,6 +1402,7 @@ type Release struct {
 	SourceCommitId  string
```

Comment:
please provide a screenshot in the PR where all the data is filled. On the current one, there are not commit IDs